<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProfileScreen</class>
 <widget class="QWidget" name="ProfileScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1440</width>
    <height>1024</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1440</width>
    <height>1024</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1440</width>
    <height>1024</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Profile Screen</string>
  </property>
  <widget class="QFrame" name="profilePanel">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>50</y>
     <width>408</width>
     <height>342</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Plain</enum>
   </property>
   <widget class="QLabel" name="profileIcon">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>40</y>
      <width>161</width>
      <height>161</height>
     </rect>
    </property>
    <property name="pixmap">
     <pixmap resource="../../Resources.qrc">:/images/light/profile-circled.svg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="loginLabel">
    <property name="geometry">
     <rect>
      <x>54</x>
      <y>210</y>
      <width>300</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>{username}</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="logoutWidget">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>20</y>
      <width>46</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>5</number>
     </property>
     <item>
      <widget class="QPushButton" name="logoutButtonIcon">
       <property name="focusPolicy">
        <enum>Qt::FocusPolicy::NoFocus</enum>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:images/light/door.svg</normaloff>:images/light/door.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="logoutText">
       <property name="text">
        <string>Выйти</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="AnimatedButton" name="overlayButton">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>20</y>
      <width>40</width>
      <height>61</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::FocusPolicy::NoFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="AnimatedButton" name="profileButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>40</y>
      <width>161</width>
      <height>161</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="fontSizeLabel">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>80</y>
     <width>181</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string>Размер шрифта:</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QSpinBox" name="fontSizeSpinBox">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>80</y>
     <width>263</width>
     <height>105</height>
    </rect>
   </property>
   <property name="minimum">
    <number>10</number>
   </property>
   <property name="maximum">
    <number>200</number>
   </property>
   <property name="value">
    <number>128</number>
   </property>
  </widget>
  <widget class="QLabel" name="fontLabel">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>240</y>
     <width>171</width>
     <height>105</height>
    </rect>
   </property>
   <property name="text">
    <string>Шрифт:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="fontComboBox">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>220</y>
     <width>261</width>
     <height>105</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Inter</string>
    </property>
   </item>
  </widget>
  <widget class="AnimatedButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>28</width>
     <height>28</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::FocusPolicy::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:images/light/out_pointer.svg</normaloff>:images/light/out_pointer.svg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>22</width>
     <height>22</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="mushroomMessage">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>650</y>
     <width>339</width>
     <height>160</height>
    </rect>
   </property>
   <property name="text">
    <string>{message}</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="themeSwitchButton">
   <property name="geometry">
    <rect>
     <x>1330</x>
     <y>0</y>
     <width>111</width>
     <height>101</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::FocusPolicy::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1440</width>
     <height>1024</height>
    </rect>
   </property>
   <property name="pixmap">
    <pixmap resource="../../Resources.qrc">:/images/light/profile_screen.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>background</zorder>
  <zorder>backButton</zorder>
  <zorder>profilePanel</zorder>
  <zorder>fontSizeLabel</zorder>
  <zorder>fontSizeSpinBox</zorder>
  <zorder>fontLabel</zorder>
  <zorder>fontComboBox</zorder>
  <zorder>mushroomMessage</zorder>
  <zorder>themeSwitchButton</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AnimatedButton</class>
   <extends>QPushButton</extends>
   <header>src/AnimatedButton/AnimatedButton.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../Resources.qrc"/>
 </resources>
 <connections/>
</ui>
